### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by puethe**
Don't understand what `DoSync.py` in the main directory is for - is this still needed or was it a temporary workaround?

### CODE REVIEW

**Comment by kathryn1995** [comment]
I think that I wrote DoSync.py as an independent script for something else and stored it here so it was under version control while it was incorporated into the main functionality of the project

File: README.md [hunk 1]
```diff
@@ -1,72 +1,47 @@
...
-- A module to upload files in the lane directory that are not easily reproducible (i.e. not .bam , .fastq.gz , \_tmp directory).
+This software backs up data produced by the Pathogen Informatics sequencing pipelines to an S3 server, thereby automatically creating missing buckets.
+It can either take a study name or a file with a list of studies as input. The pipeline data created for the chosen study/studies is traced with  [pf](https://github.com/sanger-pathogens/Bio-Path-Find) and then uploaded/synchronized with the data on the S3 server. Temporary files and files that are easily reproducible (BAM, FASTQ) are ignored to speed up the process.
 
 ## Installation
-There are a number of ways to install <software name> and details are provided below. If you encounter an issue when installing <software name> please contact your local system administrator. If you encounter a bug please log it [here](link_to_github_issues_page) or email us at path-help@sanger.ac.uk <or appropriate tool email list e.g. iva@sanger.ac.uk>.
 
 ### Required dependencies
   * Python 3.7
-  * boto3
```
Comment (kathryn1995) [inline README.md:43]: Is boto3 not required in the list because it will be installed in the setup?

Comment (puethe) [reply README.md:43]: Yes exactly. No need to preinstall it.

File: README.md [hunk 2]
```diff
@@ -1,72 +1,47 @@
 # archive_project
-Backup the pipeline results to S3
+Tool to back up the data produced by the Pathogen Informatics sequencing pipelines to an S3 server
```
Comment (kathryn1995) [inline README.md:2]: Maybe mention that it isn't all of the data, just that that can't be easily reproduced

**Comment by puethe** [comment]
> I think that I wrote DoSync.py as an independent script for something else and stored it here so it was under version control while it was incorporated into the main functionality of the project

So is it incorporated now? Would prefer to remove it unless it's still needed for something. If needed will move it to helper_scripts.

**Comment by kathryn1995** [comment]
> > I think that I wrote DoSync.py as an independent script for something else and stored it here so it was under version control while it was incorporated into the main functionality of the project
> 
> So is it incorporated now? Would prefer to remove it unless it's still needed for something. If needed will move it to helper_scripts.

It is incorporated. I think having it in helper scripts would be useful though. Thanks!

File: README.md (hunk 2 above)
Comment (puethe) [reply README.md:2]: I mention this in the "introduction". I believe that this is sufficient as a one-liner.

**Pull request merged**